packages/neuron-wallet/src/main.ts

```diff
@@ -28,15 +25,19 @@ function createWindow() {
     minHeight: 600,
     show: false,
     webPreferences: {
-      devTools: !app.isPackaged,
+      devTools: isDevelopmentMode,
     },
   })
 
   windowState.manage(mainWindow)
 
   Menu.setApplicationMenu(menu)
 
-  mainWindow.loadURL(app.isPackaged ? ENTRY.PROD : ENTRY.DEV)
+  const ENTRY = {
```

Comment:
`global` variables usually defined at the top

Reply:
I'd prefer to move it to where it's used (if there's only one place that uses it).

Reply:
In fact in this case, I believe we should simply delete `ENTRY`, and use the two URLs directly.

Reply:
OK, make it simple for now.